gapis/api/api.go

```diff
@@ -60,6 +60,9 @@ type API interface {
 
 	// GetFramegraph returns the framegraph of the capture.
 	GetFramegraph(ctx context.Context, p *path.Capture) (*Framegraph, error)
+
+	// ProfileStaticAnalysis computes the static analysis profiling data of a capture.
+	ProfileStaticAnalysis(ctx context.Context, p *path.Capture) (interface{}, error)
```

Comment:
Why is the return value untyped? How am I supposed to know what this returns when the error is nil?